recipes/perl-text-fuzzy/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.29" %}
+{% set sha256 = "3df5cfd2ca1a4c5ca7ff7bab3cc8d53ad2064e134cbf11004f3cf8c4b9055bff" %}
+
+# regex to use for Windows build hack
+{% set win_patch = "s|C:\\\\strawberry\\\\c|$ENV{LIBRARY_PREFIX}\\\\mingw-w64|g" %}
+
+package:
+  name: perl-text-fuzzy
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/B/BK/BKB/Text-Fuzzy-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # needed for linker to find some m2w64 libs
+  merge_build_host: True  # [win]
+  script:
+    - >-
+      perl Makefile.PL INSTALLDIRS=vendor NO_PERLLOCAL=1 NO_PACKLIST=1 MAKE=make
+      && (for /r . %%f in (*Makefile) do perl -i -pe "{{ win_patch }}" %%f)  # [win]
+      && make
+      && make test
+      && make install VERBINST=1
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [unix]
+    - m2w64-gcc  # [win]
+    - make
+  host:
+    - perl >=5.16
+  run:
+    - perl >=5.16
```

Comment:
```
 host:
    - perl >=5.16
  run:
    - perl >=5.16
```

For the Windows build at least, this needs to be 5.32 or else remove the version altogether (preferred). The build should get pinned to the latest Perl build in any case.

Reply:
I removed the version